docs/csharp/whats-new/csharp-7.md

```diff
@@ -386,7 +386,7 @@ when reading it the first time. These methods must be understood
 outside of the context of the single calling location.
 
 For those designs, *local functions* enable you to declare methods
-inside the context of another method. This makes it easier for readers
+inside the context of another method or inside constructor. This makes it easier for readers
```

Comment:
It's not just methods and constructors either. It's also property and event accessors, lambda expressions and anonymous methods, and finalizers. I don't think it makes sense to enumerate all those here.